adapters/criteo/criteotest/exemplary/simple-video.json

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
Would like to request to add a test case in which request is having both mediatype impressions - banner and video.

Reply:
Thanks, I added test cases with request having both mediatypes.